pkg/kubectl/cmd/exec.go

```diff
@@ -69,7 +69,7 @@ func NewCmdExec(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *
 	}
 	cmd.Flags().StringVarP(&options.PodName, "pod", "p", "", "Pod name")
 	// TODO support UID
-	cmd.Flags().StringVarP(&options.ContainerName, "container", "c", "", "Container name. If omitted, the first container in the pod will be chosen")
+	cmd.Flags().StringVarP(&options.ContainerName, "container", "c", "", "Container name. If omitted, the first user container in the pod will be chosen")
```

Comment:
In the proposal we used "app container" and "init container".  I'm not sure we want to use the terminology here though - I'd recommend just calling it "first container" until we move init containers to beta.